modules/flowable-assertions/flowable-process-assertions/src/main/java/org/flowable/assertions/process/HistoricProcessInstanceAssert.java

```diff
@@ -0,0 +1,150 @@
+package org.flowable.assertions.process;
+
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.ListAssert;
+import org.flowable.engine.ProcessEngine;
+import org.flowable.engine.history.HistoricActivityInstance;
+import org.flowable.engine.history.HistoricProcessInstance;
+import org.flowable.identitylink.api.IdentityLink;
+import org.flowable.identitylink.api.history.HistoricIdentityLink;
+import org.flowable.task.api.history.HistoricTaskInstance;
+import org.flowable.variable.api.history.HistoricVariableInstance;
+
+import static org.flowable.assertions.process.FlowableProcessAssertions.assertThat;
+import static org.flowable.assertions.process.Utils.*;
```

Comment:
Project standards do not allow for package level imports; they should be enumerated individually.

Reply:
resolved